esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -644,14 +736,19 @@ def regrid_rectilinear_to_unstructured(src_cube, mesh_cube, mdtol=0):
         target cell. ``mdtol=0`` means no missing data is tolerated while ``mdtol=1``
         will mean the resulting element will be masked if and only if all the
         overlapping cells of the ``src_cube`` are masked.
+    method : str, default="conservative"
+        The regridding method which :mod:`ESMF` uses to calculate weights. Can be
+        either "conservative" or "bilinear".
```

Comment:
Could reference the original class attributes, to make this less of a closed box, e.g. 

```
:attr:`ESMF.RegridMethod.CONSERVE`
```

Or is that overkill?